src/registrar/tests/test_models_domain.py

```diff
@@ -964,6 +978,22 @@ def test_is_disclosed_on_security_contact(self):
         # Confirm that we are getting the desired email
         self.assertEqual(domain.security_contact.email, expectedSecContact.email)
 
+    def test_disclose_security_emails(self):
+        """
+        Tests that command disclose_security_emails runs successfully with
+        appropriate EPP calll to UpdateContact.
+        """
+        domain, _ = Domain.objects.get_or_create(name="igorville.gov")
```

Comment:
igorville is a real domain name in our system now, I would avoid using it on new tests. also you would want to double check that it is in one of your two allowed states.